src/libcommon/utility/types.h

```diff
@@ -369,7 +369,7 @@ enum class AppStateKey {
 };
 std::string toString(AppStateKey e);
 
-constexpr int64_t SELF_RESTARTE_DISABLE_VALUE = -1;
+constexpr int64_t SELF_RESTARTER_DISABLE_VALUE = -1;
```

Comment:
Here is an opportunity to rename this variable with a cameCase name.

Reply:
Done